WordPress/src/main/res/values/attrs.xml

```diff
@@ -82,4 +82,8 @@
         </attr>
     </declare-styleable>
 
+    <declare-styleable name="BoundedWidth">
```

Comment:
Personal preference: I like to use the "wp" prefix for our own custom attributes to avoid possible collisions with third-party libraries (and even Android itself) down the road.